zerver/management/commands/runtusd.py

```diff
@@ -59,10 +59,10 @@ def handle(self, *args: Any, **options: Any) -> None:
             tusd_args.append(f"-s3-bucket={settings.S3_AUTH_UPLOADS_BUCKET}")
             if settings.S3_ENDPOINT_URL is not None:
                 tusd_args.append(f"-s3-endpoint={settings.S3_ENDPOINT_URL}")
-            assert settings.S3_KEY is not None
-            assert settings.S3_SECRET_KEY is not None
+            if settings.S3_KEY is not None:
+                env_vars["AWS_ACCESS_KEY_ID"] = settings.S3_KEY
+            if settings.S3_SECRET_KEY is not None:
+                env_vars["AWS_SECRET_ACCESS_KEY"] = settings.S3_SECRET_KEY
             assert settings.S3_REGION is not None
```

Comment:
This probably deserves the same treatment, right? I think we've had a few people complain they needed to set this newly in 10.x.